components/anchor/AnchorLink.tsx

```diff
@@ -77,4 +91,27 @@ export default class AnchorLink extends React.Component<AnchorLinkProps, any> {
       </div>
     );
   }
+
+  private handleClick = (e: React.MouseEvent<HTMLElement>) => {
+    const {scrollTo, onClick} = this.context.antAnchor;
+    const {href, title} = this.props;
+    if (onClick) {
+      onClick(e, {title, href});
+    }
+    scrollTo(href);
+  };
+
+  private registerLink(): void {
```

Comment:
No need to add private or public under this issue.

Reply:
This is a small fix. I thought that It would be nice because it doesn't take me a lot of time.

Reply:
Please follow the exits code style.